src/structures/BaseMessageComponent.js

```diff
@@ -79,6 +79,11 @@ class BaseMessageComponent {
         component = new MessageSelectMenu(data);
         break;
       }
+      case MessageComponentTypes.INPUT_TEXT: {
+        const FormTextInput = require('./FormTextInput');
+        component = new FormTextInput(data);
```

Comment:
Why isn't this creating a MessageForm?

Reply:
? MessageForms aren't message component types